src/layout/HeaderRow.js

```diff
@@ -28,7 +16,7 @@ class HeaderRow extends React.Component {
             <div className={classes} {...otherProps}>
                 {title ? <span className="mdl-layout-title">{title}</span> : null}
                 <div className="mdl-layout-spacer"></div>
```

Comment:
Should we replace this with the new `<Spacer />` or remove it?
I don't see any use-cases where we would not put a spacer between the title and the children/navigation...